packages/aws-rfdk/lib/core/lib/exporting-log-group.ts

```diff
@@ -126,12 +126,11 @@ export class ExportingLogGroup extends Construct {
         maxRetries: 7,
       },
     });
-    const logRetentionFunctionLogicalId = 'LogRetentionaae0aa3c5b4d4f87b02d85b201efdd8a';
-    const logRetentionFunction = Stack.of(this).node.tryFindChild(logRetentionFunctionLogicalId);
-    if (logRetentionFunction) {
-      const cfnFunction = logRetentionFunction.node.defaultChild as CfnFunction;
-      cfnFunction.addPropertyOverride('Timeout', 30);
-    }
+    // referenced from cdk code: https://github.com/aws/aws-cdk/blob/main/packages/@aws-cdk/aws-logs/lib/log-retention.ts#L123
```

Comment:
nit: this URL is not pinned to a commit, so the line number may drift when CDK's default branch pointer is modified.

The best approach is to browse the git tag for the version RFDK currently targets (`2.33.0`) and link to that.

```suggestion
    // referenced from cdk code: https://github.com/aws/aws-cdk/blob/v2.33.0/packages/@aws-cdk/aws-logs/lib/log-retention.ts#L116
```

Reply:
Will do